public/Aryan Suri/Sidebar/sidebar.js

```diff
@@ -1,6 +1,22 @@
 "use strict";
-window.addEventListener("load", Sidebar);
+window.addEventListener("load", () => {
+    if (Sidebar && !LibreTexts.active.sidebar) {
+        LibreTexts.active.sidebar = true;
+        Sidebar();
+    }
+});
+activateBeeLine();
```

Comment:
activateBeelLine() should be handled by Mathjax's initialization and shouldn't need a secondary initialization here